minimal-kernel/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Jupyter Development Team.
+FROM debian:jessie
+
+MAINTAINER Jupyter Project <jupyter@googlegroups.com>
+
+USER root
+
+# Install all OS dependencies for fully functional notebook server
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -yq --no-install-recommends \
```

Comment:
Probably can axe wget, bzip2, unzip, sudo. Maybe ca-certificates too but not sure. Try the seed URL option on kernel gateway and see if it can fetch a notebook from an HTTPS endpoint properly.